website/src/pages/auth/user-auth-layout.tsx

```diff
@@ -165,4 +110,52 @@ export const FormFields = ({
   )
 }
 
-export const Page = LoginPage
+interface FormSubmitButtonType {
+  form: unstable_FormStateReturn<any | undefined>
+  label: string
+}
+
+export const FormSubmitButton = ({ form, label }: FormSubmitButtonType) => {
+  return (
+    <div className={positionButton}>
+      <FormSubmit {...form} as={Button} className={loginButton}>
+        {label}
+      </FormSubmit>
+    </div>
+  )
+}
+
+export const ResetLink = () => {
+  const token = useCredentials()
+
+  return (
+    <>
+      {!token && (
+        <label>
+          Forgot your password?{" "}
+          <Link href="/auth/reset-password">Reset password</Link>
+        </label>
+      )}
+    </>
+  )
+}
+
+export const SignupLink = () => {
+  return (
+    <>
+      <label>
+        Dont have an account? <Link href="/auth/signup">Sign Up</Link>
```

Comment:
I'm not sure `label` is semantically correct here. I think `label` is just for form inputs.

Reply:
So, according to the actual HTML specs, this is technically an appropriate use of label. That being said, Web Content Accessibility Guidelines (WCAG) 2.0 would disallow using `label` here. I will certainly follow the more restrictive guidelines here, especially since accessibility is impacted. Change also made for `ResetLink` and `LoginLink`.